### PR TITLE
[Android] News new content behaviour fix

### PR DESCRIPTION
**Pull request by tapanmodh**
Resolves https://github.com/brave/brave-browser/issues/22182
Resolves https://github.com/brave/brave-browser/issues/22183
Resolves https://github.com/brave/brave-browser/issues/22356

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Review by alexsafe — APPROVED** [review]

File: android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java [hunk 1]
```diff
@@ -888,11 +876,12 @@ private void initNews() {
                 if (mActivity == null) {
                     mActivity = BraveActivity.getBraveActivity();
                 }
-                if (!ConfigurationUtils.isLandscape(mActivity) && prevScrollPosition > 1) {
+                /*if (!ConfigurationUtils.isLandscape(mActivity) && prevScrollPosition > 1) {
```
Comment (deeppandya) [inline android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java:879]: Do we need commented out code here ?

**Review by deeppandya — APPROVED** [review]
LGTM

File: android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java (hunk 1 above)
Comment (tapanmodh) [reply android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java:879]: Not needed. I'll remove it

**Comment by kjozwiak** [comment]
Verification **PASSED** on both `Pixel 7` & `Samsung S10+` running `Android 12` using the following build(s):

```
Brave | 1.40.12 Chromium: 101.0.4951.41 (Official Build) canary (32-bit)
--- | ---
Revision | 93c720db8323b3ec10d056025ab95c23a31997c9-refs/branch-heads/4951@{#904}
OS | Android 12; Build/SP1A.210812.016
```

#### `Test Case #1` - https://github.com/brave/brave-browser/issues/22182 & https://github.com/brave/brave-browser/issues/22183

**`Example of issue fixed on Pixel 6`**

https://user-images.githubusercontent.com/2602313/165215179-d658d7b6-82b7-4316-a8e9-847d97be831e.mp4

**`Example of issue fixed on Samsung S10+`**

https://user-images.githubusercontent.com/2602313/165215211-007688ef-49da-4f93-8b6c-06eef4263121.mp4

**Pull request merged**